### ISSUE TITLE
Repo should be renamed to e.g. "reward_distances"

### ISSUE THREAD

**Issue by anshradh** [HumanCompatibleAI/epic#2]
Since the implementation for divergence-free reward distance calculations will also live in this repo (as well as possible other reward function distance implementations), the repo should be renamed to reflect that. `setup.py` etc. will also need to be modified.

**Comment by Rocamonde** [comment]
@erik @AdamGleave what's your view on this?

**Comment by AdamGleave** [comment]
Yeah, EPIC is definitely too restrictive. We were having a discussion on Slack for what to call these class of distances with @ejnnr and @JoarSkalse. I think we're currently leaning towards STARTC (STAndardised Reward Comparison) which is also available on PyPi. Full options:

> ELMs (Epic-Like Metrics)
> IRC metrics (Informative Reward Comparison)
> NORC (NOrmalised Reward Comparison)
> STARC (STAndardised Reward Comparison)
> EFARC (Epic-FAmily Reward Comparison)

**Comment by Rocamonde** [comment]
I like starc!
